debian10/revisions.bzl

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is autogenerated by the dependency update service and should not be modified directly.
+# For more details, check the deps_spec.yaml file in the current folder.
+
+DEBIAN10_TAR = struct(
+    revision = "1569803469",
+    sha256 = "5fa8836c3f7a574577907e267173ed96528dbd1c1b1145879029c55370e7aed2",
```

Comment:
You probably know this, but you can set the `revision` and `sha256` to some invalid strings not to get confused why this won't work. Or you can even delete this struct as it can't be used. This file doesn't exist in GCS until FUS runs first. After FUS pushes that file to GCS, DUS will be able to update these pins for you.